src/App.vue

```diff
@@ -4,18 +4,28 @@ import SidePanel from '@/components/home/SidePanel.vue'
 import { RennesApp } from '@/services/RennesApp'
 import MapComponent from '@/components/map/MapComponent.vue'
 import mapConfig from './map.config.json'
+import { useViewsStore } from './stores/views'
 import UiButtonWithTooltip from '@/components/ui/UiButtonWithTooltip.vue'
 
+import { viewList } from './model/views.model'
+
+const viewStore = useViewsStore()
+
 onBeforeMount(() => {
   const rennesApp = new RennesApp(mapConfig)
   provide('rennesApp', rennesApp)
 })
+
+function isLeftPanelRetractable() {
+  const retractableList = [viewList['map-pcaet']]
+  return retractableList.includes(viewStore.currentView)
+}
 </script>
 
 <template>
   <main class="h-screen flex">
     <aside class="z-10 absolute">
-      <SidePanel :is-retractable="false">
+      <SidePanel :is-retractable="isLeftPanelRetractable()">
```

Comment:
hi @MaYohann Yes. I plan to move the side panel to the common UI. I will move it when we merge this one, to avoid confusion (e.g. I import from common UI, but it's not there yet).